scripts/quarantine.yaml

```diff
@@ -125,3 +125,9 @@
   platforms:
     - all
   comment: "ANT samples fail to build; need to change main()'s signature to int main(void)"
+
+- scenarios:
+    - sample.bluetooth.mesh.sensor_client
+  platforms:
+    - nrf52dk_nrf52832
+  comment: "Sensor Client sample doesn't fit to RAM"
```

Comment:
Suggestion implies that we are disabling the test because of we want to save on time on CI. I suggest the comment as it is, or 'Disabling the Sensor Client sample because of RAM usage issue' ?